samples/net/mdns_responder/src/service.c

```diff
@@ -121,7 +121,12 @@ void service(void)
 			log_strdup(addrstr), ntohs(*portp));
 
 		/* send a banner */
-		welcome(client_fd);
+		r = welcome(client_fd);
+		if (r == -1) {
+			NET_DBG("send() failed (%d)", errno);
```

Comment:
I'd say this should be NET_ERR().

Reply:
@pfalcon Should I also add `NET_ERR()` when `bind()`, `listen()` and `socket()` fail?

Reply:
Yeah I used `NET_DBG()` since it's used everywhere else. The main focus is to fix the coverity issue where the return value was not checked. I could add a separate commit to change `NET_DBG()` to `NET_ERR()` on top of the current changes.

@jukkar @tbursztyka what are your thoughts on the matter?

Reply:
> The main focus is to fix the coverity issue

Right, I remember that, and why I say I don't want to introduce more scope creep into that. LGTM.